tox.ini

```diff
@@ -122,3 +122,10 @@ changedir={toxinidir}/lib/authorization/candig_authz_service/
 commands=
     pip install git+https://github.com/candig/candig_authz_service.git
     screen -dmS {envname} candig_authz_service --host {env:AUTHORIZATION_SERVICE_HOST} --port {env:AUTHORIZATION_SERVICE_PORT}
+
+[testenv:opa]
+commands=
+    mkdir -p {toxinidir}/bin/
+    curl -Lo {toxinidir}/bin/opa https://openpolicyagent.org/downloads/latest/opa_linux_amd64
+    chmod 755 {toxinidir}/bin/opa
+    screen -dmS {envname} {toxinidir}/bin/opa run -s {toxinidir}/lib/opa/opa_service/metadata-service.json {toxinidir}/lib/opa/opa_service/metadata-service.rego
```

Comment:
@afrosimon I am just curious about `scren`, what is it? terminal multiplexer?

Reply:
Yup pretty much a terminal multiplexer